### DISCUSSION TITLE
请问ios版本是不能用了么？

### DISCUSSION THREAD

**Discussion by cyrilpetrilli** [getlantern/lantern-forum-cn#675] [General]
ios版本的app 没有办法找到授权代码  而且也连接不上  是我的问题还是普遍的问题？ 有解决的时间表么？

**Comment by MaxMKMax** [comment]
@cyrilpetrilli 您好. 下周应该新版本会上线了. 就可以解决这个问题了
谢谢您的耐心等待. 不好意思让您久等了!
